.codegen/error_mapping.go.tmpl

```diff
@@ -0,0 +1,39 @@
+// Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+package apierr
+
+import (
+	"errors"
+    "os"
+    "context"
+)
+
+{{$builtinErrors := dict 
+    "BAD_REQUEST" "os.ErrInvalid"
+    "DEADLINE_EXCEEDED" "os.ErrDeadlineExceeded"
+    "CANCELLED" "context.Canceled"
+    "NOT_FOUND" "os.ErrNotExist"
+    "UNAUTHENTICATED" "os.ErrPermission"
+    "PERMISSION_DENIED" "os.ErrPermission"
+    "RESOURCE_CONFLICT" "os.ErrExist"
+}}
```

Comment:
I think we should leave these out. If a client wants to adapt Databricks errors to another error library, they should wrap those errors themselves. For instance, there is ErrExist in os and in io/fs, why only extend the os version?